src/main/scala/io/github/pauljamescleary/petstore/infrastructure/repository/doobie/DoobieUserRepositoryInterpreter.scala

```diff
@@ -45,6 +45,14 @@ class DoobieUserRepositoryInterpreter[F[_]: Monad](val xa: Transactor[F])
       case None =>
         none[User].pure[F]
     }
+
```

Comment:
The other functions here do not use `override`, should remove the `override` for consistency